### PR TITLE
[master]Add docs for the reserve flag

### PR DESCRIPTION
**Pull request by changzhi1990**
Docs for the reserved flag in zvmsdk for get_volume_connector method.

### CODE REVIEW

File: doc/source/parameters.yaml [hunk 1]
```diff
@@ -1183,3 +1183,9 @@ skipzipl:
   in: body
   required: false
   type: bool
+fcp_reserve:
+  description: |
+    If or not reserve the FCP devices.
+  in: path
```
Comment (bjhuangr) [inline doc/source/parameters.yaml:1189]: in path or request body?

Comment (changzhi1990) [reply doc/source/parameters.yaml]: body. I have uploaded a new pr.

**Comment by dongyanyang** [comment]
rebase

**Comment by dongyanyang** [comment]
@bjhuangr @jichenjc pls help to merge. The API change code has been merged in master and 1.5.n several days ago.

**Pull request merged**